integration-tests/src/test/java/com/example/AbstractComponent.java

```diff
@@ -3,7 +3,7 @@
 import dagger.Component;
 
 @Component
-abstract class AbstractComponent {
+public abstract class AbstractComponent {
```

Comment:
Visibility now validated earlier and needed to trigger actual desired validation failure